### PR TITLE
tests: Add tests for `go/vt/external`

### PR DESCRIPTION
**Pull request by Maniktherana**
## Description
Bumps coverage from 15.4% to 61.5%

Question: some functions require a `db *sql.DB` as an argument. Not sure how to approach testing these. Do I make a sqlite DB to test them or use [sqlmock](https://github.com/DATA-DOG/go-sqlmock)?

## Related Issue(s)
#14931

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [ ] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on CI?
-   [x] Documentation was added or is not required

### CODE REVIEW

**Comment by frouioui** [comment]
> some functions require a `db *sql.DB` as an argument. Not sure how to approach testing these. Do I make a sqlite DB to test them or use [sqlmock](https://github.com/DATA-DOG/go-sqlmock)?

We usually have E2E tests for these, perhaps we already have for these functions/features actually. But coverage is not enabled yet for E2E tests (https://github.com/vitessio/vitess/issues/14932)

File: go/vt/external/golib/sqlutils/sqlutils_test.go [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type any = interface{}
+
```
Comment (frouioui) [inline go/vt/external/golib/sqlutils/sqlutils_test.go:28]: Why not dropping this type alias and directly using the Golang builtin `any`?

Comment (frouioui) [reply go/vt/external/golib/sqlutils/sqlutils_test.go:28]: the builtin does the exact same thing:
```go
// any is an alias for interface{} and is equivalent to interface{} in all ways.
type any = interface{}
```

Comment (Maniktherana) [reply go/vt/external/golib/sqlutils/sqlutils_test.go:28]: Ahh, didn't know that was a thing in go

**Review by frouioui — APPROVED** [review]
Besides the comments I left it looks good to me.

File: go/vt/external/golib/sqlutils/sqlutils_test.go (hunk 1 above)
Comment (Maniktherana) [reply go/vt/external/golib/sqlutils/sqlutils_test.go]: fixed

**Review by GuptaManan100 — APPROVED** [review]
LGTM!

**Pull request merged**